package.json

```diff
@@ -65,5 +65,6 @@
       "eslint --fix",
       "git add"
     ]
-  }
-}
+  },
+  "repository": "git@github.com:galacean/runtime.git"
```

Comment:
Why add this line?

Reply:
Specify the place where your code lives. This is helpful for people who want to contribute. If the git repo is on GitHub, then the npm docs command will be able to find you.

Reply:
Our package is galacean/engine